### PR TITLE
Run python scripts in the correct directory

### PR DESCRIPTION
**Pull request by GTcreyon**
Currently, python scripts are found and run in whatever directory the patcher is executed from. This changes that, so that scripts are found in the patch folder and run in the decomp folder.

### CODE REVIEW

File: flash_patcher/parse/visitor/stage_visitor.py [hunk 1]
```diff
@@ -59,9 +60,13 @@ def visitPythonFile(
         example output: "DoAction1.as,DoAction2.as"
         Python script names may not include spaces.
         """
+        cwd = Path.cwd()
```
Comment (rayyaw) [inline flash_patcher/parse/visitor/stage_visitor.py]: Would it be possible to move this into a helper function in the `util` folder? If so we should do that, like how we did with the safe file handling.

**Review by rayyaw — CHANGES_REQUESTED** [review]
Some minor fixes required, but overall everything looks good.

File: flash_patcher/parse/visitor/stage_visitor.py (hunk 1 above)
Comment (GTcreyon) [reply flash_patcher/parse/visitor/stage_visitor.py]: Done!

**Review by rayyaw — APPROVED** [review]
LGTM. Sending back to Creyon for final review.

**Pull request merged**